src/particles/ReferenceParticle.H

```diff
@@ -160,6 +160,23 @@ namespace impactx
             return *this;
         }
 
+        /** Get reference particle magnetic rigidity
+         *
+         * @returns magnetic rigidity Brho in T-m
```

Comment:
Would this be a bit clearer for the unit?
```suggestion
         * @returns magnetic rigidity Brho in T*m
```